src/main/java/core/basesyntax/FileWork.java

```diff
@@ -16,6 +16,9 @@
  */
 
 public class FileWork {
+
+    private final char symbolThatAllWordsStartWith = 'w';
```

Comment:
this is a constant
what are checkstyle requirements for constants? should they be static or not? should names be written in lower or upper case?